surveys/serializers.py

```diff
@@ -6,6 +7,16 @@
 from . import models
 
 
+def update_nested(d, u):
+    for k, v in u.items():
+        if isinstance(v, collections.Mapping):
+            r = update_nested(d.get(k, {}), v)
+            d[k] = r
+        else:
+            d[k] = u[k]
+    return d
```

Comment:
First thing I see in this PR is this method...

![http://s2.quickmeme.com/img/83/83206ae67da70662c246b9cca0b92d2da77cae199a14e8d23e102739d5192b2e.jpg](http://s2.quickmeme.com/img/83/83206ae67da70662c246b9cca0b92d2da77cae199a14e8d23e102739d5192b2e.jpg)